src/main/java/com/robinpowered/sdk/model/Identifier.java

```diff
@@ -72,6 +72,9 @@ public static Interface fromString(String value) {
     private final String value;
     private final DateTime createdAt;
 
+    // Mutable
+    private Integer deviceId;
```

Comment:
@christophermark An example of this is in the `Presence` model, which requires a `@Nullable Integer spaceId`